plugins/base/src/main/kotlin/translators/documentables/DefaultPageCreator.kt

```diff
@@ -299,6 +299,23 @@ open class DefaultPageCreator(
             }
         }
 
+        fun DocumentableContentBuilder.contentForSamples() {
+            val samples = tags.withTypeNamed<Sample>()
+            if (samples.isNotEmpty()) {
+                platforms.forEach { platformData ->
+                    val content = samples.filter { it.value.isEmpty() || platformData in it.value }
+                    if (content.isNotEmpty()) {
+                        group(platformData = setOf(platformData)) {
```

Comment:
I'd give the `ContentGroup` a `ContentKind.Sample` kind

Reply:
Done